.github/ISSUE_TEMPLATE/test-plan.md

```diff
@@ -0,0 +1,62 @@
+## Test plan
+
+## Feature
+
+Name:
```

Comment:
consider linking to original issue instead, where this can be quickly referenced  (and/or use the title